src/Gateway.php

```diff
@@ -11,46 +16,46 @@
  */
 class Gateway extends AbstractGateway
 {
-    public function getName()
+    public function getName(): string
     {
         return 'PayZen';
     }
 
-    public function getDefaultParameters()
+    public function getDefaultParameters(): array
     {
-        return array(
+        return [
             'certificate' => '',
             'testMode' => false,
-        );
+        ];
     }
 
     public function getCertificate()
```

Comment:
Return type and i saw that we have it on many other function that doesn't have it (not changed by you)?

Reply:
yes, it's because some of the functions are from parent classes and I can't change these ones